index.html

```diff
@@ -61,11 +66,12 @@ <h1 class="logo">My Logo</h1>
        <section class="parting">
         <div class="tonic">
           <div class="tonicing">
-            <img src="images/snapshot1.png" alt="Tonic Work" />
+            <img class="pic-1" src="images/snapshot1.png" alt="Tonic Work" />
+            <img class="pic-2" src="images/Snapshoot Portfolio.png" alt="Tonic Work" />
           </div>
           <div class="tonicContent">
             <h2>Tonic</h2>
-            <div class="tonicList1">
+            <div class="tonicListR">
               <ul>
                 <li class="licap">CANOPY</li>
                 <li>• Back End Dev</li>
```

Comment:
Good job, but please make sure to use the same font weight, and the same spacing. It'll increase your design quality.

![image](https://user-images.githubusercontent.com/34779928/194647802-fa805595-0140-4efb-bfd4-c5f8ca0dc4bb.png)